tests/coalaTest.py

```diff
@@ -323,6 +323,15 @@ def test_coala_without_color(self):
             self.assertEqual(
                 retval, 0, 'coala must return zero when there are no errors')
 
+    def test_format_and_json(self):
+        retval, stdout, stderr = execute_coala(coala.main, 'coala', '--json',
+                                               '--format', '-b',
+                                               'SpaceConsistencyTestBear',
+                                               '-f', '*.py'
+                                               )
```

Comment:
Move the closng brackets to the previous line

Reply:
No, do not do that.  Instead, add a trailing comma on the previous line